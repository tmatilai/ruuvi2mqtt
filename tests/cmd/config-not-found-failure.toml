bin.name = "ruuvi2mqtt"
args = "--config non-existing.yaml"
status.code = 1
