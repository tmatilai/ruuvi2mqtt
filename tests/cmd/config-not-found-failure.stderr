[[..] INFO  ruuvi2mqtt] ruuvi2mqtt [..]
Error: Failed to read non-existing.yaml

Caused by:
    No such file or directory (os error 2)
